themes-default/slim/static/js/templates/app-header.vue

```diff
@@ -13,24 +13,26 @@
         </div>
         <div v-if="auth.isAuthenticated" class="collapse navbar-collapse" id="main_nav">
             <ul class="nav navbar-nav navbar-right">
-                <li id="NAVhome" class="navbar-split dropdown" :class="{ active: topMenu === 'home' }">
-                    <app-link href="home/" class="dropdown-toggle" aria-haspopup="true" data-toggle="dropdown" data-hover="dropdown"><span>Shows</span>
-                    <b class="caret"></b>
+                <li @mouseover="toggleDropdown('home')" @mouseout="toggleDropdown('home')" id="NAVhome" class="navbar-split dropdown" :class="{ active: topMenu === 'home' }">
```

Comment:
Just a thought: Would using the `.passive` modifier on all these events perform better?